src/map/lua/lua_baseentity.cpp

```diff
@@ -664,6 +664,28 @@ void CLuaBaseEntity::setVolatileCharVar(std::string const& varName, int32 value,
     }
 }
 
+/************************************************************************
+ *  Function: getAllLocalVars()
+ *  Purpose : Returns all variables assigned locally to an entity
+ *  Example :
+ *  Notes   :
+ ************************************************************************/
+
+auto CLuaBaseEntity::getAllLocalVars() -> sol::table
```

Comment:
Make these function calls consistant with GetLocalVars

Reply:
> Make these function calls consistant with GetLocalVars

Thanks, I updated the PR!